scale/ingest/test/triggers/test_ingest_recipe_handler.py

```diff
@@ -38,7 +38,7 @@ def setUp(self):
         self.source_file.add_data_type_tag('type3')
         manifest = job_test_utils.create_seed_manifest(inputs_files=[{'name': 'INPUT_FILE', 'media_types': ['text/plain'], 'required': True, 'multiple': True}], inputs_json=[])
         self.jt1 = job_test_utils.create_seed_job_type(manifest=manifest)
-        recipe_type_def = {'version': '6',
```

Comment:
Shouldn't we duplicate the tests for 6 and 7 so that we are confident we have consistency between them? At some point 7 will drift on past what 6 supports.

Reply:
Added another type definition to test both 6 and 7.